src/query/interpreter.cpp

```diff
@@ -3108,6 +3124,9 @@ void Interpreter::Commit() {
   if (!commit_confirmed_by_all_sync_repplicas) {
     throw ReplicationException("At least one SYNC replica has not confirmed committing last transaction.");
   }
+
+  memgraph::metrics::IncrementCounter(memgraph::metrics::CommitedTransactions);
```

Comment:
Commit function is not noexcept so I would definitely put those two under utils::OnScopeExit